.github/workflows/build_and_test.yml

```diff
@@ -42,3 +42,32 @@ jobs:
             --packages-up-to obstacle_stop_planner_refine \
             --event-handlers console_cohesion+ \
             --return-code-on-test-failure
+
+  code-metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Measure code metrics
+        run: |
+          git clone https://github.com/terryyin/lizard.git
+          mkdir public
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
+            --CCN 20 --length 200 --arguments 6 \
+            --html > public/lizard_only_wo_test.html || true
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
+            --CCN 20 --length 200 --arguments 6 || true
```

Comment:
Why do you run this twice? To print in the terminal?

Reply:
There are two reasons:
- To check the result in the terminal.
- To get the Return Code. (With the --html option, the return code will always be 0)